### PR TITLE
new task-bash.json file added for git bash

### PR DESCRIPTION
**Pull request by Mahdi-Hasan**
This json file is tested with git bash and working fine.

### CODE REVIEW

File: Config-Files/C&C++/Windows/tasks.json [hunk 1]
```diff
@@ -1,54 +1,59 @@
...
+          "<",
+          "input.txt",
+          ">",
+          "output.txt",
+          "&&",
+          "rm",
+          "${objectFile}",
+          "&&",
+          "rm",
+          "${fileBasename}",
```
Comment (jspw) [inline Config-Files/C&C++/Windows/tasks.json]: did you just removed basefile ? @Mahdi-Hasan

**Comment by jspw** [comment]
may be `Mahdi-Hasan:new-feature/windows-bash` sounds good insted of `Mahdi-Hasan:bugfix/windows-bash` as its not a bug fix.

File: Config-Files/C&C++/Windows/tasks.json [hunk 2]
```diff
@@ -51,4 +51,4 @@
       }
     }
   ]
-}
\ No newline at end of file
+}
```
Comment (jspw) [inline Config-Files/C&C++/Windows/tasks.json:54]: and its better to avoid this type of changes like adding a new line in file :3 @Mahdi-Hasan

**Comment by jspw** [comment]
Currently i dont have a windows machine to test the task.json file for bash, so pr merge can take sometimes.

File: Config-Files/C&C++/Windows/tasks.json (hunk 1 above)
Comment (Mahdi-Hasan) [reply Config-Files/C&C++/Windows/tasks.json]: The base file was copied to working directory. I just removing the copied file from working directory. But the main file is remined in its own folder. @jspw

**Pull request closed without merging**